### ISSUE TITLE
Imports components removed in Ember Beta and Canary

### ISSUE THREAD

**Issue by gilest** [adopted-ember-addons/ember-keyboard#491]
In `ember-keyboard-first-responder-inputs.js`:

```js
import TextArea from '@ember/component/text-area';
import TextField from '@ember/component/text-field';
```

Looks like these components were removed from `ember-source` in [this commit](https://github.com/emberjs/ember.js/commit/5d24ae7480aae886d7a526d15332f53fe5b1689c#diff-f2e5c2ee12b2a561981417a8a936ad956278298dc38b343200e6a7f9970a20dd).

I'm not sure what the best approach is. Could perhaps use `ember-compatability-helpers` to exclude the initializer from builds in ember gte 4.0. 

This will fail test builds even when the initializer is disabled in the dummy app. Eg: [ember-file upload builds](https://github.com/adopted-ember-addons/ember-file-upload/actions/runs/1457823246)

**Comment by SergeAstapov** [comment]
@gilest see my comment [in #489](https://github.com/adopted-ember-addons/ember-keyboard/pull/489#issuecomment-968399835).

TL;DR there does not seem simple option to make it work with Ember v4.

**Comment by gilest** [comment]
Thank you @SergeAstapov – I can specifically include that package in dev builds internally. Godspeed with v7.

**Comment by SergeAstapov** [comment]
@gilest just had an idea: we may have v7.beta-0 including dropping support for Node.js 10 and #489.

Then we could upgrade `ember-cli-addon-docs` to use `ember-keyboard@v7.beta-0`. This will make CI happy for addons using `ember-cli-addon-docs` in their `devDependencies`.

**Comment by gilest** [comment]
@SergeAstapov That's true and addon authors could pin the beta right away anyway. See my scuffed fix in adopted-ember-addons/ember-file-upload#603 😅

**Comment by SergeAstapov** [comment]
@gilest we could make a new release of `ember-cli-addon-docs` with `ember-keyboard@v7.beta-0` so addons could enjoy green CI again

**Comment by gilest** [comment]
@SergeAstapov That would certainly be helpful :)
